styles.css

```diff
@@ -84,18 +96,60 @@ h1 {
   margin-right: 140px;
 }
 
-.sid-cons .material-symbols-outlined {
-  display: none;
-}
-
-.toolbar {
-  border-radius: 20px;
-  background-color: #fff;
-}
-
 /* start of media query */
 
 @media (max-width: 470px) {
+  .nav-menu {
```

Comment:
- [ ] [Optional] Kindly make sure that your mobile menu is responsive and follows the design from the chosen template in Figma (e.g., font, colors, images, tests, margins), your page is not responsive you can use the ```%``` and ```vw``` to specify the width and height of the page and menu
- [ ] [Optional] You can modify the close button so it is as the provided design
- [ ] [Optional] The background in the Figma design has a blur,  please add this feature to your design so it can match the Figma one

![Screenshot_20220512_080421](https://user-images.githubusercontent.com/25227022/168011388-e5a08d1e-327b-49ca-8b0e-1bdde660d61f.png)